### PR TITLE
Add english strings for monthly screening reports

### PR DESCRIPTION
**Pull request by olttwa**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1019,6 +1019,40 @@
...
+  <string name="month_short_jun">Jun</string><!-- June -->
+  <string name="month_short_jul">Jul</string><!-- July -->
+  <string name="month_short_aug">Aug</string><!-- August -->
+  <string name="month_short_sep">Sep</string><!-- September -->
+  <string name="month_short_oct">Oct</string><!-- October -->
+  <string name="month_short_nov">Nov</string><!-- November -->
+  <string name="month_short_dec">Dec</string><!-- December -->
+
+  <!--Ethiopian Calendar Month short forms-->
+  <string name="month_short_mes">Mes</string><!-- Meskerem -->
```
Comment (olttwa) [inline app/src/main/res/values/strings.xml:1037]: Should these 13 months be in English file or a separate Ethiopia file? It doesn't make sense to translate them to Hindi, Bangla, Punjabi, etc.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1019,6 +1019,40 @@
...
+  <string name="month_short_gin">Gin</string><!-- Ginbot -->
+  <string name="month_short_sen">Sen</string><!-- Sene -->
+  <string name="month_short_ham">Ham</string><!-- Hamle -->
+  <string name="month_short_neh">Neh</string><!-- Nehasa -->
+  <string name="month_short_pag">Pag</string><!-- Pagume -->
+
   <!--Monthly screen reports-->
+  <string name="monthly_screening_reports">Monthly screening reports</string>
+  <string name="monthly_screening_reports_report">report</string><!--Jan 2023 report-->
+  <string name="monthly_screening_reports_screening_report">screening report</string><!--Jan 2023 screening report-->
```
Comment (siddh1004) [inline app/src/main/res/values/strings.xml:1054]: @olttwa Could not find the usage for this string. Can you correct me, if I'm missing something.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1019,6 +1019,40 @@
...
+  <string name="month_short_apr">Apr</string><!-- April -->
+  <string name="month_short_may">May</string><!-- May -->
+  <string name="month_short_jun">Jun</string><!-- June -->
+  <string name="month_short_jul">Jul</string><!-- July -->
+  <string name="month_short_aug">Aug</string><!-- August -->
+  <string name="month_short_sep">Sep</string><!-- September -->
+  <string name="month_short_oct">Oct</string><!-- October -->
+  <string name="month_short_nov">Nov</string><!-- November -->
+  <string name="month_short_dec">Dec</string><!-- December -->
+
```
Comment (siddh1004) [inline app/src/main/res/values/strings.xml:1035]: Conversion for English Calendar Month short forms are not required, as they are already handled by the date formatter.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1019,6 +1019,40 @@
...
+  <string name="month_short_tah">Tah</string><!-- Tahsas -->
+  <string name="month_short_tir">Tir</string><!-- Tir -->
+  <string name="month_short_yak">Yak</string><!-- Yakatit -->
+  <string name="month_short_mag">Mag</string><!-- Maggabit -->
+  <string name="month_short_miy">Miy</string><!-- Miyazya -->
+  <string name="month_short_gin">Gin</string><!-- Ginbot -->
+  <string name="month_short_sen">Sen</string><!-- Sene -->
+  <string name="month_short_ham">Ham</string><!-- Hamle -->
+  <string name="month_short_neh">Neh</string><!-- Nehasa -->
+  <string name="month_short_pag">Pag</string><!-- Pagume -->
```
Comment (siddh1004) [inline app/src/main/res/values/strings.xml:1049]: Conversion for Ethiopian Calendar Month short forms are not required, as they are already handled by the date formatter.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1019,6 +1019,40 @@
...
+
   <!--Monthly screen reports-->
+  <string name="monthly_screening_reports">Monthly screening reports</string>
+  <string name="monthly_screening_reports_report">report</string><!--Jan 2023 report-->
+  <string name="monthly_screening_reports_screening_report">screening report</string><!--Jan 2023 screening report-->
   <string name="monthly_screening_reports_submit_btn_text">Submit</string>
+  <string name="monthly_screening_reports_submit_report">Submit report</string>
+  <string name="monthly_screening_reports_submitted">Submitted</string><!--Will also be used for Submitted Jan 27, 06:24pm-->
 </resources>
```
Comment (siddh1004) [inline app/src/main/res/values/strings.xml:1058]: @olttwa We have missed the translations for https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8/Simple-(Production)?node-id=28784%3A55073&t=PMQ7dXr2x7RtrgKv-4 and https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8/Simple-(Production)?node-id=29202%3A49821&t=PMQ7dXr2x7RtrgKv-4. Can you add them as well?

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (olttwa) [reply app/src/main/res/values/strings.xml:1054]: It's on the Questionnaire input page

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (olttwa) [reply app/src/main/res/values/strings.xml:1035]: Added them. Please take a look.

**Review by siddh1004 — APPROVED** [review]

**Review by harimohanraj89 — COMMENTED** [review]
This PR adds strings, but should it also _use_ them somewhere?

**Pull request closed without merging**